images/Dockerfile.ubuntu.certified.pushpin

```diff
@@ -1,27 +1,18 @@
-FROM ubuntu:16.04
+FROM harishanand95/pushpin
 MAINTAINER Harish Anand "https://github.com/harishanand95"
 
-RUN apt-get update --fix-missing
-RUN apt-get install -y git
-RUN apt-get install -y pkg-config qtbase5-dev libzmq3-dev mongrel2-core zurl
-RUN git clone git://github.com/fanout/pushpin.git && \
-cd pushpin  && \
-git checkout develop  && \
-git submodule init && git submodule update  && \
-./configure --qtselect=5 && \
-make && \
-make install && \
-mkdir /root/pushpin && \
-mkdir /root/pushpin/runner && \
-mkdir /root/pushpin/runner/certs
-
+ENV LOGNAME=nobody
```

Comment:
Pushpin fails when LOGNAME env is not set.
The new Pushpin image will contain the pushpin's installation steps that were part of the old Dockerfile.
Issue: #24